electron-builder.yml

```diff
@@ -75,12 +80,13 @@ linux:
     Name: Heroic Games Launcher
     Comment[de]: Ein Open Source Spielelauncher for GOG und Epic Games
   asarUnpack:
-    - build/bin/linux/legendary
-    - build/bin/linux/gogdl
-    - build/bin/linux/nile
-    - build/bin/linux/vulkan-helper
+    - build/bin/x64/linux/legendary
+    - build/bin/x64/linux/gogdl
+    - build/bin/x64/linux/nile
+    - build/bin/x64/linux/vulkan-helper
   files:
-    - build/bin/linux/*
+    - build/bin/x64/linux/*
+    - build/bin/arm64/linux/*
```

Comment:
do we need to add `bin/arm64/linux` paths in the `asarUnpack` declaration before this?

Reply:
Oh, not sure why I added this path. We don't have arm64 Linux binaries
Will remove it instead. Once we do have those binaries, a PR adding them to the download script would then also add this path again

Reply:
Ended up modifying the pattern to include both paths, didn't think about using a globstar for the arch part of the path before